### ISSUE TITLE
Cannot Join the host game

### ISSUE THREAD

**Issue by IvyWillowTheBat** [DeadlySurprise/FO2-Splitscreen#18]
Whenever I get the games running, they cannot find games within LAN. I've tried setting up a game on either side, and searching but flatout comes back with no results.

**Comment by MorMund** [comment]
Hi
* Are you running any other mods for FlatOut?
* Which Version of the game do you own? (Steam, CD and which Language)
* Lastly could post the output of running `ipconfig` in the Windows command prompt?

**Comment by robertgroh** [comment]
Hi, 
I also can't get find a LAN-Game in the second instance.

- No mods installed (freshly installed through Steam (Steam.dll in the same folder))
- Steam-Distribution (Language: German)
- ```ipconfig``` doesn't list any virtual network interface and my connections don't use the address range of the virtual network according to the ```Splitscreen_Settings.xml```
```ipconfig```-Output (unfortunately in german):
```
C:\Users\Robert>ipconfig

Windows-IP-Konfiguration

Ethernet-Adapter LAN-Verbindung:
   Medienstatus. . . . . . . . . . . : Medium getrennt
   Verbindungsspezifisches DNS-Suffix: fritz.box

Drahtlos-LAN-Adapter LAN-Verbindung* 5:
   Medienstatus. . . . . . . . . . . : Medium getrennt
   Verbindungsspezifisches DNS-Suffix:

Drahtlos-LAN-Adapter LAN-Verbindung* 6:
   Medienstatus. . . . . . . . . . . : Medium getrennt
   Verbindungsspezifisches DNS-Suffix:

Drahtlos-LAN-Adapter Drahtlosnetzwerkverbindung:
   Verbindungsspezifisches DNS-Suffix: fritz.box
   IPv6-Adresse. . . . . . . . . . . : 2001:16b8:2add:4f00:5c7d:4c08:3ea0:6f47
   Temporäre IPv6-Adresse. . . . . . : 2001:16b8:2add:4f00:8b2:35da:5886:bc55
   Verbindungslokale IPv6-Adresse  . : fe80::5c7d:4c08:3ea0:6f47%11
   IPv4-Adresse  . . . . . . . . . . : 192.168.178.76
   Subnetzmaske  . . . . . . . . . . : 255.255.255.0
   Standardgateway . . . . . . . . . : fe80::ca0e:14ff:fec2:e687%11
                                       192.168.178.1

Ethernet-Adapter Bluetooth-Netzwerkverbindung:
   Medienstatus. . . . . . . . . . . : Medium getrennt
   Verbindungsspezifisches DNS-Suffix:
```

![FO2_Splitscreen_No_Games_found](https://user-images.githubusercontent.com/5773739/62421547-c4d92380-b6a3-11e9-82ab-b45eb54a28d2.png)

---
*Edit/Update*
It works fine with an old DRM-free CD-Version of Flatout 2 on the same PC. With the CD-Version, I got a Permission-Dialog of the Windows-Firewall, which i accepted. Therefore I tried to set Firewall-Exceptions for the Steam-Version (which were already set) and the `Lauchner.exe`, but it didn't help for the Steam-Version

**Comment by nastynav36** [comment]
Any update on this particular issue? Having the exact same problem of not being able to join the host game. Using the Steam version.

**Comment by MorMund** [comment]
@nastynav36 I tested it with the steam version of the game and can confirm that it is broken at the moment. So I should be able to figure out what's wrong soonish™.

**Comment by nastynav36** [comment]
One of my favorite games from back in the the day. Looks like a fantastic mod. Thanks for your work!

**Comment by MorMund** [comment]
This issue is resolved with #21. The bug fix will be included in the next version (0.1.2).
Thank you @robertgroh for providing detailed debug info and @nastynav36 for confirming it.

Edit: The new version is now [available](https://github.com/DeadlySurprise/FO2-Splitscreen/releases/tag/v0.1.2).
